phpBB/phpbb/textformatter/s9e/quote_helper.php

```diff
@@ -40,6 +40,7 @@ class quote_helper
 	public function __construct(\phpbb\user $user, $root_path, $php_ext)
 	{
 		$this->post_url = append_sid($root_path . 'viewtopic.' . $php_ext, 'p={POST_ID}#p{POST_ID}', false);
+		$this->msg_url = append_sid($root_path . 'ucp.' . $php_ext, 'i=pm&mode=view&p={MSG_ID}', false);
```

Comment:
Don't forget to add it to the class as a protected property.

Reply:
Done